synapse/lib/link.py

```diff
@@ -104,6 +104,8 @@ async def __anit__(self, reader, writer, info=None):
 
         self.iden = s_common.guid()
 
+        writer._transport.set_write_buffer_limits(0)
```

Comment:
Per @Cisphyx This was added to ensure that we were writing data out as soon as possible based on https://vorpus.org/blog/some-thoughts-on-asynchronous-api-design-in-a-post-asyncawait-world/#bug-3-closing-time
This was due to the fact that we were missing data in the links from the spawned process to the calling axon.